improver_tests/acceptance/test_threshold.py

```diff
@@ -41,114 +41,57 @@
 run_cli = acc.run_cli(CLI)
 
 
-def test_basic(tmp_path):
-    """Test basic thresholding"""
-    kgo_dir = acc.kgo_root() / "threshold/basic"
+@pytest.mark.parametrize(
+    "extra_args,kgo_subdir",
```

Comment:
I'm not sure that we've made such large scale use of the parametrize functionality in the acceptance tests. I'm not sure if there might be readability concerns, although we do do things like this in the unit tests, I think. Do you have any thoughts?

Reply:
I like it as I am more confident that the things that should be consistent between tests and the things that should change are clearly separated.